other/block_stale_images/block-stale-images.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: block-stale-images
+  annotations:
+    policies.kyverno.io/title: Block Stale Images
+    policies.kyverno.io/category: Other
+    policies.kyverno.io/severity: medium
+    kyverno.io/kyverno-version: 1.6.0
+    policies.kyverno.io/minversion: 1.6.0
+    kyverno.io/kubernetes-version: "1.23"
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+      Images that are old usually have some open security vulnerabilities which are not patched.
+      This policy checks the contents of every container image and inspects them for the create time.
+      If it finds any image which was built more than 6 months ago this policy blocks the deployment.
+spec:
+  validationFailureAction: audit 
+  rules:
+    - name: block-stale-images
+      match:
+        resources:
```

Comment:
Please add an `any` clause under `match` here to conform to how match and exclude is standardizing.